README.md

```diff
@@ -543,3 +572,44 @@ Nexus instead. The modified files will be accessible at the
 again in a future request, it will use it directly from Nexus rather than downloading it and
 uploading it again. This guarantees that any dependency used for a Cachito request can be used again
 in a future Cachito request.
+
+### pip
+
+The pip package manager works by parsing the `requirements.txt` and `requirements-build.txt` files
+present in the source repository to determine what dependencies are required to build the
+application. It is possible to specify different file path(s) for the requirements files as long
+as the files use the expected format.
+
+Cachito then creates two repositories in an instance of Nexus it manages that contain just the
+dependencies discovered in the requirements files. PyPI dependencies are uploaded to a PyPI hosted
+repository, external dependencies are uploaded to a raw repository. Connection information for the
+hosted repository is provided as the `PIP_INDEX_URL` environment variable accessible at the
+`/api/v1/requests/<id>/environment-variables` endpoint. To make external dependencies available,
+Cachito modifies the requirements files for the request by replacing relevant entries with their
+corresponding URLs from the raw repository. The modified requirements files are accessible at the
+`/api/v1/requests/<id>/configuration-files` endpoint.
+
+Cachito will produce a bundle that is downloadable at `/api/v1/requests/<id>/download`. This
+bundle will contain the application source code in the `app` directory and individual source
+archives of all the dependencies in the `deps/pip` directory. These archives are not meant to be
+used to build the application. They are there for convenience so that the dependency sources can be
+published alongside your application sources. In addition, they can be used to to install packages
+directly from the filesystem with `pip install --no-index --no-deps <path/to/archive>` in the event
+that the application needs to be built without Cachito and the Nexus instance it manages.
```

Comment:
Is this different from python being able to install in an offline mode?

Reply:
Hmm, true, this is a way to install in offline mode (but, when building with OSBS, it would require changes in the Dockerfile). Then again, for npm we have this, which would also suggest the possibility:
> In addition, they can be used to populate a local npm
registry in the event that the application needs to be built without Cachito and the Nexus instance
it manages.

@athos-ribeiro @lcarva what exactly do we mean by "Offline Installations"?

Reply:
if the `gomod-vendor` flag is the golang's version of offline installation, then that still requires a change as well... just not to the Dockerfile.

Reply:
Offline installation means that you can take the tarball created by Cachito and use those dependencies without any network connection at all. This is true for gomod. It's not for NPM because network access to the Nexus repo is required. 

In either case, the Cachito tarball always includes all sources. This is to fulfill source compliance requirements. It could also be used to perform offline operations, but that would require changes to the pkg manager commands to use them directly.

Reply:
Would it be fair to say that offline installations for gomod "just work" (i.e. the user does not have to change anything) while npm and pip would require significant changes in the user's workflow to install offline? Therefore we say that only gomod supports offline installations?

Reply:
@arewm I updated the Offline Installations section to reflect what I've learned here, let me know if it's more clear now (thanks for the review BTW)

Reply:
Thanks, looks good.